packages/server/src/db/invite-codes.ts

```diff
@@ -0,0 +1,31 @@
+import { Entity, Column, PrimaryColumn } from 'typeorm'
+
+@Entity({ name: 'invite_codes' })
+export class InviteCode {
+  @PrimaryColumn('varchar')
+  code: string
+
+  @Column('integer')
+  availableUses: number
+
+  @Column({ type: 'boolean', default: false })
+  disabled: boolean
+
+  @Column('varchar')
+  forUser: string
+
+  @Column('varchar')
+  createdBy: string
+}
+
+@Entity({ name: 'invite_codes_uses' })
+export class InviteCodeUses {
+  @PrimaryColumn('varchar')
+  code: string
+
+  @PrimaryColumn('varchar')
+  usedBy: string
+
+  @PrimaryColumn('varchar')
```

Comment:
Is there a reason `usedAt` oughtta be part of the primary key?  It seems like it might be fair to limit each entry's identifier to just `(code, usedBy)`.

Reply:
oh nope you're totally right about that. thanks for the catch